x-pack/plugins/upgrade_assistant/server/lib/es_deprecations_status.ts

```diff
@@ -15,85 +15,127 @@ import {
   convertFeaturesToIndicesArray,
 } from './es_system_indices_migration';
 
+export async function getHealthIndicators(
+  dataClient: IScopedClusterClient
+): Promise<EnrichedDeprecationInfo[]> {
+  const healthIndicators = await dataClient.asCurrentUser.healthReport();
+
+  // Temporarily ignoring due to untyped ES indicators
+  // types will be available during 8.9.0
```

Comment:
Should we open up an issue to track this follow-up work?

Reply:
@alisonelizabeth i'll open one in kibana, we have one already here for the follow up work https://github.com/elastic/elasticsearch-specification/pull/2146